### PR TITLE
Add colopl logos to index page

### PR DESCRIPTION
**Pull request by it2911**
I add colopl logo to the index page.

**Please confirm with me before you merge it.**

### CODE REVIEW

**Comment by chengchen4pingcap** [comment]
@YiniXu9506 @queenypingcap

File: src/pages/jp/index.js [hunk 1]
```diff
@@ -19,11 +19,13 @@ import Button from '../../components/button'
...
 
 const caseLogos = [
-  'unext',
   'square',
+  'unext',
+  'colopl-ja',
   'lenovo',
   'tencent',
   'bigo',
+  'colopl-ja',
```
Comment (YiniXu9506) [inline src/pages/jp/index.js:28]: Do you need to add two same colopl logos on this page?

Comment (chengchen4pingcap) [reply src/pages/jp/index.js]: Fixup

**Review by YiniXu9506 — APPROVED** [review]
LGTM

**Comment by queenypingcap** [comment]
LGTM

**Pull request merged**